src/pages/UDT/state.ts

```diff
@@ -19,3 +19,11 @@ export const defaultUDTInfo: UDT = {
     hashType: '',
   },
 }
+
+export const defaultOmigaInscriptionInfo: OmigaInscription = {
+  ...defaultUDTInfo,
+  mintStatus: MintStatus.Closed,
+  mintLimit: '0',
+  expectedSupply: '0',
+  inscriptionId: '',
```

Comment:
`inscription id` is specific for `inscription item`(as implied in the document), naming it `id` or `inscriptionInfoId` would be clear

Reply:
Updated.